core/parallel_state_processor.go

```diff
@@ -797,6 +795,10 @@ func (p *ParallelStateProcessor) Process(block *types.Block, statedb *state.Stat
 		}
 	}
 
+	if txDAG != nil && txDAG.Type() == types.EmptyTxDAGType {
+		return nil, nil, 0, FallbackToSerialProcessorErr
+	}
+
```

Comment:
will this cause the code at line 752 and 755 (DAOFork and PreContract) run twice if fallback?